activejob-web.gemspec

```diff
@@ -1,26 +1,28 @@
-require_relative "../lib/activejob/web/sion"
+# frozen_string_literal: true
+
+require_relative './lib/activejob/web/version'
 
 Gem::Specification.new do |spec|
-  spec.name        = "activejob-web"
+  spec.name        = 'activejob-web'
   spec.version     = Activejob::Web::VERSION
-  spec.authors     = ["mohammednazeer"]
-  spec.email       = ["mohammednazeer@mallow-tech.com"]
-  spec.homepage    = "TODO"
-  spec.summary     = "TODO: Summary of Activejob::Web."
-  spec.description = "TODO: Description of Activejob::Web."
-  spec.license     = "MIT"
+  spec.authors     = ['gowtham']
+  spec.email       = ['gowtham.kuppusamy@mallow-tech.com']
```

Comment:
@GowthamK01 Please append your name instead of replacing

Reply:
@gomathi-mallow Have appended my name instead of replacing here